evap/static/scss/components/_grid.scss

```diff
@@ -122,7 +132,30 @@
     }
 }
 
-@each $col in name, semester, responsible, participants, result {
+.textanswer-review-grid {
+    @extend %grid-row;
```

Comment:
This seems a bit sus, the grid container specializes the "grid row" concept :thinking: I suppose it's an artifact of us using one grid container per row before? I am also not sure which rules we need exactly from that, `display: grid` for sure and maybe the `media-breakpoint` business, but the other stuff is overwritten anyways.

Maybe this PR isn't the right place to dig into this and find clarity, but if we are aiming for CSS grid to replace more tables on the site, we should disentangle this at some point.